test/linalg/eigen.jl

```diff
@@ -105,6 +105,13 @@ end
     end
 end
 
+let a = rand(10, 10)
+    f = eigfact(a)
+    sort!(f)
```

Comment:
probably `f = sort(eigfact(a))` instead to make sure that both `sort` and `sort!` are tested.